code/modules/cooking/recipes/recipes_breads.dm

```diff
@@ -171,6 +171,24 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/stuffing
 
+/singleton/recipe/angry_bread
+	appliance = OVEN
+	fruit = list("carrot" = 1)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/flatbread,
+		/obj/item/reagent_containers/food/snacks/xenomeat,
+		/obj/item/reagent_containers/food/snacks/cheesewedge
+	)
+	result = /obj/item/reagent_containers/food/snacks/angry_bread
+
+/singleton/recipe/angry_bread/make_food(obj/container) // removing some reagents instead of using RECIPE_REAGENT_REPLACE so anything that's in the original meat you use is still in the final dish (namely whether or not the meat you use has polytrinic acid).
+	. = ..()
+	var/list/results = .
+	for(var/thing in results)
+		var/obj/item/XMG = thing
```

Comment:
No all-uppercase variable names (like `XMG` here). Reason is consistency, vars should have all-lowercase names, and all-uppercase names are reserved for macros.

Reply:
Though, I'm pretty sure this XMG var could just be removed and the loop defined like `for(var/obj/item/thing in results)`